test_scripts/API/OpenMenuRPC/008_Open_Menu_RPC_without_policy_SystemContext_ALERT_DISALLOWED.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0116-open-menu.md
+-- Description:
+-- In case:
+-- 1) Mobile application is set to appropriate HMI level and System Context ALERT
+-- 2) ShowAppMenu RPC is not allowed by policy
+-- 3) Mobile sends ShowAppMenu request without menuID parameter to SDL
+-- SDL does:
+-- 1) not send ShowAppMenu request to HMI
+-- 2) send ShowAppMenu response with resultCode = DISALLOWED to mobile
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/OpenMenuRPC/commonOpenMenuRPC')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.appHMIType = { "PROJECTION" }
+
+--[[ Local Variables ]]
+local resulCode = "DISALLOWED"
+
+
+--[[ Local Function ]]
+local function pTUpdateFunc(pTbl)
+  pTbl.policy_table.functional_groupings["Base-4"].rpcs.ShowAppMenu = nil
+  pTbl.policy_table.functional_groupings["Base-4"].rpcs["Alert"].hmi_levels = {
+    "FULL",
+    "BACKGROUND",
+    "LIMITED"
+  }
+end
+
+local function sendAlertSuccess()
+  common.getMobileSession():SendRPC("Alert", {
+    alertText1 = "a",
+    alertText2 = "1",
+    alertText3 = "_",
+    ttsChunks = {
+      {
+        text = "TTSChunk",
+        type = "TEXT"
+      }
+    },
+    duration = 6000
+  })
+  local AlertId
+  EXPECT_HMICALL("UI.Alert", {
```

Comment:
@TMyza, use common.getHMIConnection():ExpectRequest(...) instead of EXPECT_HMICALL(...)